src/BenchmarkDotNet.Core/Diagnosers/CopiedDataContracts.cs

```diff
@@ -53,6 +53,8 @@ public class DisassembledMethod
         public string Problem { get; set; }
 
         public Map[] Maps { get; set; }
+
+        public string CommandLine { get; set; }
```

Comment:
the file is called `CopiedDataContracts` and it starts with `// keep it in sync with src\BenchmarkDotNet.Disassembler.x64\DataContracts.cs!`

could you please add the same field to the `src\BenchmarkDotNet.Disassembler.x64\DataContracts.cs` file?

Reply:
@adamsitnik should I fill this field? If so, I can suggest something like CommandLine = Environment.CommandLine in both DisassembleMethod and CreateEmpty.

Reply:
@morgan-kn no, just add it to the class. 

Some background: Windows disassembler must not have any dependencies to BDN, but it serializes the result to file. BDN has to be able to deserialize it, and also have no dependencies to the disassembler. Hence these two projects must duplicate same data contract, and we need this lame, manual files sync ;/

Reply:
@adamsitnik thank you =)

Reply:
@adamsitnik, why do we have to sync these files manually?
In the `BenchmarkDotNet.Disassembler.x86` we use linked files:
```xml
<Compile Include="..\BenchmarkDotNet.Disassembler.x64\DataContracts.cs"
         Link="DataContracts.cs" />
```
What do you think about using the same trick for `BenchmarkDotNet.Core`?

Reply:
@AndreyAkinshin I remember that I have tried this approach, but for some reason, it did not work. I am not sure what was the problem that I hit back then ;/ It's most probably worth to give it a try once again